src/transformState.js

```diff
@@ -62,7 +64,25 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const key in transforms) {
```

Comment:
transforms - це масив, тому ітеруй циклом`for of` 
```suggestion
  for (const transforms of transforms) {
```